lib/src/testdata/testSimpleBB.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
```

Comment:
move this up a dir.  this dir should not contain executable code

Reply:
Moved up a dir.